zeus/client.py

```diff
@@ -23,14 +23,17 @@
 
 class ZeusClient(MetricsInterface, LogsInterface, AlertsInterface,
                  TrigAlertsInterface):
-
     """Zeus Client class, implementing wrapper methods for the
     Zeus API.
 
     """
 
     def __init__(self, user_token, server):
         self.token = user_token
```

Comment:
they says `user_token` but this was `external_token`. header change is necessary to support for `user token`, meanwhile `external token` is used in url path.
 

```python
def __init__(self, user_token, server):
    # => user_token is eiter external token or user token... variable name is a bit misleading,,,
    pass
```